### PR TITLE
Add PrivateKey authentication for TransIP

### PR DESCRIPTION
**Pull request by blackshadev**
Add PrivateKey authentication for TransIP.

Appearently the AccessTokens have limited lifetime. TransIP has a separate authentication method for it: with your AccountName and a PrivateKey.
This PR adds that capability while still providing the AccessToken authentication method as well.

### CODE REVIEW

File: docs/_providers/transip.md [hunk 1]
```diff
@@ -9,7 +9,21 @@ jsId: TRANSIP
 
 ## Configuration
 
-In your providers config json file you must include a TransIP personal access token:
+In your providers config json file you must include your TransIP credentials
+
+You can login with your AccountName and a PrivateKey which can be generated in the TransIP control panel. The PrivateKey is a stringified version of the private key given by the API, see the example below , each newline is replaced by "\n".
```
Comment (tlimoncelli) [inline docs/_providers/transip.md:14]: No space before comma.

**Review by tlimoncelli — APPROVED** [review]

**Comment by tlimoncelli** [comment]
Please rebase.

**Comment by blackshadev** [comment]
> Please rebase.

done

**Comment by blackshadev** [comment]
@tlimoncelli I rebased as you asked, could you merge this? I would like to use it with you DNScontrol Docker image.

**Comment by tlimoncelli** [comment]
Actually... since the delay was my fault, I've rebased it in https://github.com/StackExchange/dnscontrol/pull/1212 and will merge it once the tests complete.

**Comment by tlimoncelli** [comment]
I've merged https://github.com/StackExchange/dnscontrol/pull/1212 and I'm closing this one. 

Thanks for adding this feature!

**Pull request closed without merging**